readthedocs/builds/models.py

```diff
@@ -217,6 +217,11 @@ def is_private(self):
             return self.project.external_builds_privacy_level == PRIVATE
         return self.privacy_level == PRIVATE
 
+    @property
+    def is_public(self):
+        """Check if the version is public (taking external versions into consideration)."""
```

Comment:
Does this code take external versions into consideration? It doesn't seem to. A clearer docstring would be useful here:

```
Check if the version is public.

Default implementation just checks that it isn't private.
**Downstream implementations should check for...**

Reply:
I understand the check for external version is inside `is_private` method at this line https://github.com/readthedocs/readthedocs.org/pull/10067/files#diff-2026582e018afc479271c2c856ebf5163912b778923ab381526f03d3aa2363a3R217. Does it make sense?

Reply:
is_private checks for external versions, this is just negating that.

Reply:
That feels non-obvious to me. When I read the code, I figured `is_private` was just a model field or similar, not a property. But I understand how it works, but could likely be made more explicit/clear.

Reply:
```suggestion
        """Check if the version is public."""
        # ``is_private`` understands both normal and external versions
```